src/storage/src/test_util/config_util.rs

```diff
@@ -62,14 +64,16 @@ pub async fn new_store_config_and_region_map(
     builder.root(store_dir);
     let object_store = ObjectStore::new(builder).unwrap().finish();
 
-    new_store_config_with_object_store(region_name, store_dir, object_store).await
+    new_store_config_with_object_store(region_name, store_dir, object_store, max_files_in_level_0)
+        .await
 }
 
 /// Create a new StoreConfig with given object store.
 pub async fn new_store_config_with_object_store(
     region_name: &str,
     store_dir: &str,
     object_store: ObjectStore,
+    max_file_in_l0: Option<usize>,
```

Comment:
What about taking `EngineConfig` as an argument? So we could override other options if needed.
```suggestion
    config: EngineConfig,
```